.github/workflows/release-confidential-identity.yml

```diff
@@ -2,8 +2,8 @@ name: Upload Confidential Identitiy Release Asset
 
 on:
   push:
-    tags:
-    - 'confidential-identity-v*'
+    branches:
+      - 'main'
```

Comment:
this `main` branch doesn't seem to exist yet.

Reply:
oops, yeah I was testing the deployment in another repo which had `main` instead of `master`. I'll change it.